scripts/verify-source-map.js

```diff
@@ -429,6 +429,8 @@ async function check(kind, testCase, toSearch, { flags, entryPoints, crlf, follo
     }
 
     if (!failed) removeRecursiveSync(tempDir)
+
+    outMap.destroy()
```

Comment:
Looks like it may be better to use `.with` so you don't need to remember to use `.destroy`.  https://www.npmjs.com/package/source-map#sourcemapconsumerwith

Reply:
Good call. Updated: [`2e52fcd` (#10)](https://github.com/cypress-io/esbuild/pull/10/commits/2e52fcd377583ae93e11714b48ccd953a5cc51e8)